namespaces/exec.go

```diff
@@ -73,7 +71,11 @@ func Exec(args []string, env []string, console string, command *exec.Cmd, contai
 	if err != nil {
 		return terminate(err)
 	}
-	defer cgroupManager.Destroy()
+	defer func() {
+		if err != nil {
```

Comment:
Shouldn't we always call cgroups destroy irrespective of err state on exiting this function?